### ISSUE TITLE
Need to set `futurehostile` to false in order to enable

### ISSUE THREAD

**Issue by wbyoung** [jshint/jshint#2425]
This doesn't really make all that much sense to me & the documentation says to set enforcing options to `true`. This may be the intended behavior or may have become the case when `futurehostile` was changed to an enforcing option in #2216.

**Comment by lukeapage** [comment]
@jugglinmike do you have a comment on this? it does seem weird to me..

**Comment by jugglinmike** [comment]
Yes, this is not ideal. As mentioned in the referenced pull request, we want to enable this behavior by default and expose it as a relaxing option. We really should have an issue to track that intention, so we can use this for that purpose.
